CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+
+**Take a look at this [issue about our README](https://github.com/alan-turing-institute/tric-dt/issues/17) for an example of how we work with issues :rocket:**
+
+### Open a new issue
+If you'd like to start a conversation about something which isn't already an open issue, you'd love you to [open a new one](https://github.com/alan-turing-institute/tric-dt/issues/new/choose)! 
+
+## Pull requests (PR)
+If you spot something in these materials which you want to suggest changing directly, we invite you to [submit a pull request (PR)](https://cassgvp.github.io/github-for-collaborative-documentation/docs/tut/3-2-fork-their-repo.html)! 
+
+If it's something simple like fixing a typo (there may be typos!), you are welcome to go straight  into a pull request :blush:. 
+
+If your suggested change is more substantial, please [open an issue](#open-a-new-issue) for discussion before getting too deep into the work, or better yet, [contact us](#contact-us)! 
+
+A discussion before submitting a PR makes a space ensure your suggestions can be integrated with work that's currently underway, and that everyone is on the same page with the goal of the work you plan to carry out.
+
+## Particpate in our activities
+
+### Stakeholder map
+We are creating a map of the TRIC-DT community to understand the shape and structure of our community, in terms of expertise, engagement and opportunities. The map will inform the development of effective and systematic engagement strategies, and support members of the community in connecting with eachother. An interactive visualisation of individuals (name, email and affiliations), research projects and companies or institutions who are connected to or engaged with the TRIC-DT programme will be created and shared using kumu.
+
+You will soon be able to [review our stakeholder map and add themselves](https://github.com/alan-turing-institute/tric-dt/issues/46). Follow the [stakeholder map issues](https://github.com/alan-turing-institute/tric-dt/issues?q=is%3Aopen+is%3Aissue+label%3Astakeholder-map) for updates on this work.
+
+### Seminar series
+Join us online for our [next seminar](https://github.com/alan-turing-institute/tric-dt/tree/main/Seminars#attend-a-seminar) to hear about the latest foundational and applied research in digital twinning being conducted at Turing. These seminars are an excellent platform to understand the multidisciplinary challenges of Digital Twin research as each seminar brings together two speakers from distinct research domains. We encourage you to share your viewpoints and [suggest new topics](https://github.com/alan-turing-institute/tric-dt/issues/new?assignees=&labels=0%2F+idea&projects=&template=topic-suggestion-for-tric-dt-event.md&title=%5BTopic%5D)!
+
+## Code of conduct
+* Placeholder - what CoC should we invoke and how do we implement it? Preference to follow a Turing wide process*
```

Comment:
Should we push with a placeholder here? I think it'd be good to settle this before final merge